contrib/kube-prometheus/jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```diff
@@ -145,6 +146,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
       local proxySelfMetrics =
         container.new('kube-rbac-proxy-self', $._config.imageRepos.kubeRbacProxy + ':' + $._config.versions.kubeRbacProxy) +
         container.withArgs([
+          '--logtostderr',
```

Comment:
For https://github.com/brancz/kube-rbac-proxy/ would it make sense to add this flag as a default to the Dockerfile?

Reply:
Good idea, that should work. Is it a common thing to do when using glog? Not sure if in a future release we would get rid of glog entirely?

Reply:
I'm not a huge fan of adding flags into entrypoints, it gives the notion of this being the default for the binary when it's not ... we made this mistake in kube-state-metrics and are stuck with it